tests/drivers/clock_control/clock_pwm/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_PRINTK=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_LOG=y
+CONFIG_LOG_PRINTK=y
+CONFIG_LOG_MODE_IMMEDIATE=y
+CONFIG_PWM_LOG_LEVEL_DBG=y
```

Comment:
How come all of these Kconfigs are enabled? Shouldn't it be sufficient to enable `CONFIG_CLOCK_CONTROL` by default?

Reply:
thanks, yes, I removed the print options and just left CONFIG_CLOCK_CONTROL and  ZTEST.